internal/tiltfile/helm.go

```diff
@@ -103,3 +111,37 @@ func unableToFindHelmErrorMessage() error {
 
 	return fmt.Errorf("Unable to find Helm installation. Make sure `%s` is on your $PATH.", binaryName)
 }
+
+func localSubchartDependenciesFromPath(chartPath string) ([]string, error) {
+	requirementsPath := filepath.Join(chartPath, "requirements.yaml")
+	_, err := os.Stat(requirementsPath)
+	var deps []string
+	if os.IsNotExist(err) {
+		return deps, nil
+	}
+
+	dat, err := ioutil.ReadFile(requirementsPath)
+	if err != nil {
+		return deps, err
+	}
+
+	return localSubchartDependencies(dat)
+}
+
+func localSubchartDependencies(dat []byte) ([]string, error) {
+	var deps []string
+	var metadata chart.Metadata
```

Comment:
have you considered just writing a new struct with the fields you want to grab?

it seems weird to import Helm just to get 2 fields from a struct